cmd/main.go

```diff
@@ -91,6 +91,7 @@ var Options struct {
 	HWValidatorConfig           hardware.ValidatorCfg
 	JobConfig                   job.Config
 	InstructionConfig           hostcommands.InstructionConfig
+	OperatorsConfig             operators.Options
```

Comment:
Why was this added here? If you're not adding `envconfig` to `operators.Options` then you should just be able to instantiate on in `main`

Reply:
Consistent with other component configuration that are defined under Options